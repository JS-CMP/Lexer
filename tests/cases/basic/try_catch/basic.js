try {
    console.log("In try block");
} catch (e) {
    console.log("In catch block");
}
