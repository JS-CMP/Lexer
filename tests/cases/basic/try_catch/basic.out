#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {try {console[u"log"](JS::Any(u"In try block"));}catch (JS::Any& e) {console[u"log"](JS::Any(u"In catch block"));};}
